README.md

```diff
@@ -1,123 +1,129 @@
-# Feathub
-
-Feathub is a feature store that facilitates feature development and deployment
-to achieve the following objectives:
-- **Reduce duplication of data engineering efforts** by allowing new ML projects
-  to reuse and share a library of curated production-ready features already
-  registered by existing projects in the same organization.
-- **Simplify feature management** by allowing users to specify feature
-  definitions and feature processing jobs as code using a declarative framework.
-- **Facilitate feature development-to-deployment iteration** by allowing users
-  to use the same declarative feature definitions across training and serving,
-  online and offline, without training-serving skew. Feathub takes care of
-  compiling feature definitions into efficient processing jobs and executing those
-  jobs in a distributed cluster.
-
-Feathub provides SDK and infra that enable the following capabilities:
-- Define feature-view (a group of related features) as transformations and joins
-  of the existing feature-views and data sources.
-- Register and retrieve feature-views by names from feature registry.
-- Transform and materialize features for the given time range and/or keys from the
-  feature view into feature stores, by applying transformations on source
-  dataset with point-in-time correctness.
-- Fetch online features by joining features from online feature store with
-  on-demand transformations.
-
-## Architecture and Concepts
-
-<img src="docs/figures/architecture_1.png" width="50%" height="auto">
-
-The figure above shows the key components and concepts in the FeatHub
-architecture.
-
-<img src="docs/figures/architecture_2.png" width="70%" height="auto">
-
-The figure above shows how a developer can use FeatHub to facilitate feature
-engineering for model training and model inference. Please checkout [Feathub
-architecture and concepts](docs/architecture.md) for more details.
-
-## Getting Started
+FeatHub is a stream-batch unified feature store that simplifies feature
```

Comment:
Should we rename all the `Feathub` to `FeatHub`? Or we can create an issue to do that late.

Reply:
Thanks for the suggestion. It is fixed now.